app/src/main/java/org/mozilla/focus/utils/OnSwipeListener.java

```diff
@@ -14,6 +14,12 @@
 
     void onSwipeDown();
 
+    void onLongPress();
+
+    boolean onDoubleTap();
+
+    boolean onSingleTapConfirmed();
+
     class OnSwipeListenerAdapter implements OnSwipeListener {
```

Comment:
Consider using `default` modifier in `OnSwipeListener` methods so we don't need this Wrapper.